### PR TITLE
Filter out window.FT errors

### PR DESCRIPTION
**Pull request by alexflorisca**
🐿 v2.5.16

### CODE REVIEW

File: browser/bundles/o-errors.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ oErrors.init({
 	tags: {
 		appName: appInfo.name
 	},
-	filterError: function () {
-		return !window.FT.disableOErrors;
+	filterError: function (reportedObject) {
+		return !(window.FT.disableOErrors || reportedObject.error.toString().search('window.FT') > 0);
```
Comment (wheresrhys) [inline browser/bundles/o-errors.js]: maybe that's a little too loose. Would it be better to check for `window.FT.flags || window.FT.nUi`?

**Comment by alexflorisca** [comment]
@wheresrhys How about now? I think there's more places where that errors shows up. I've seen it in `.flags`, `.nUi`, `.ftNextUi` and possibly others.

**Comment by wheresrhys** [comment]
It *shoudl* just be those three values though. Would prefer to scope it more strictly for now as we don't want it to swallow errors we might not have considered and that might be _real_

**Comment by alexflorisca** [comment]
@wheresrhys Ok how about now?

**Comment by wheresrhys** [comment]
cool ta

**Comment by sjparkinson** [comment]
image

👏

**Pull request merged**